### PR TITLE
Preloaders

### PR DESCRIPTION
**Pull request by szymborski**

### CODE REVIEW

File: src/renderer/components/preloaderBar.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      type: Boolean,
+      default: false
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  .preloader-bar {
+    content: "";
```
Comment (Argeento) [inline src/renderer/components/preloaderBar.vue]: Why is the `content` property here?

File: src/renderer/components/preloaderBar.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    background-size: 600% 600%;
+
+    &--main {
+      height: 7px;
+      position: fixed;
+    }
+
+    animation: AnimationName 3s ease infinite;
+
+    @-webkit-keyframes AnimationName {
```
Comment (Argeento) [inline src/renderer/components/preloaderBar.vue]: We dont need prefixes if we only have one environment.

File: src/renderer/components/preloaderBar.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        background-position: 0% 50%
+      }
+      50% {
+        background-position: 100% 50%
+      }
+      100% {
+        background-position: 0% 50%
+      }
+    }
+    @keyframes AnimationName {
```
Comment (Argeento) [inline src/renderer/components/preloaderBar.vue]: Rename it to something that more accurately reflects the animation.  Use kebab-case.

File: src/renderer/store/modules/boards.js [hunk 4]
```diff
@@ -22,9 +22,11 @@ const mutations = {
 }
 
 const actions = {
-  async fetchBoards ({ commit }) {
+  async fetchBoards ({ commit, dispatch }) {
+    dispatch('wait/start', 'boardsLoading', { root: true })
```
Comment (Argeento) [inline src/renderer/store/modules/boards.js:26]: Maybe we should try the [waitFor decorator](https://github.com/f/vue-wait#waitforloader-string-func-function-forcesync--false)?

Comment (szymborski) [reply src/renderer/store/modules/boards.js:26]: Tried, but the problem is as always in "this" context inside waitFor callback.

**Review by Argeento — APPROVED** [review]

**Pull request merged**